src/babel/util.js

```diff
@@ -102,7 +102,7 @@ export function arrayify(val: any, mapFn?: Function): Array {
   return [val];
 }
 
-export function booleanify(val: any): boolean {
+export function booleanify(val: any): any {
```

Comment:
It should actually be removed since functions that don't have a return type are implicitly `any`.